usaspending_api/common/query_with_filters.py

```diff
@@ -893,34 +900,40 @@ def generate_elasticsearch_query(cls, filter_values: List[str], query_type: _Que
 
 
 class QueryWithFilters:
-    filter_lookup = {
-        _Keywords.underscore_name: _Keywords,
-        _Description.underscore_name: _Description,
-        _KeywordSearch.underscore_name: _KeywordSearch,
-        _TransactionKeywordSearch.underscore_name: _TransactionKeywordSearch,
-        _TimePeriods.underscore_name: _TimePeriods,
-        _AwardTypeCodes.underscore_name: _AwardTypeCodes,
-        _Agencies.underscore_name: _Agencies,
-        _RecipientSearchText.underscore_name: _RecipientSearchText,
-        _RecipientId.underscore_name: _RecipientId,
-        _RecipientScope.underscore_name: _RecipientScope,
-        _RecipientLocations.underscore_name: _RecipientLocations,
-        _RecipientTypeNames.underscore_name: _RecipientTypeNames,
-        _PlaceOfPerformanceScope.underscore_name: _PlaceOfPerformanceScope,
-        _PlaceOfPerformanceLocations.underscore_name: _PlaceOfPerformanceLocations,
-        _AwardAmounts.underscore_name: _AwardAmounts,
-        _AwardIds.underscore_name: _AwardIds,
-        _ProgramNumbers.underscore_name: _ProgramNumbers,
-        NaicsCodes.underscore_name: NaicsCodes,
-        PSCCodes.underscore_name: PSCCodes,
-        _ContractPricingTypeCodes.underscore_name: _ContractPricingTypeCodes,
-        _SetAsideTypeCodes.underscore_name: _SetAsideTypeCodes,
-        _ExtentCompetedTypeCodes.underscore_name: _ExtentCompetedTypeCodes,
-        _DisasterEmergencyFundCodes.underscore_name: _DisasterEmergencyFundCodes,
-        _QueryText.underscore_name: _QueryText,
-        _NonzeroFields.underscore_name: _NonzeroFields,
-        _ProgramActivities.underscore_name: _ProgramActivities,
-    }
+
+    @property
+    def filter_lookup(self) -> Dict[str, _Filter]:
```

Comment:
Won't block over this but as of Python 3.9 the use of `dict` is preferred over `Dict` (see [here](https://docs.python.org/3/library/typing.html#typing.Dict)).